draft-ietf-dnsop-svcb-https.md

```diff
@@ -508,13 +508,16 @@ procedure:
    acceptable parameters.  This TargetName and SvcParams represent the
    preferred endpoint.  If connection to this endpoint fails, the
    client MAY try to connect using values from a lower-priority record.
-   If all attempts fail, clients SHOULD go to step 5.
+   If all attempts fail, clients SHOULD go to step 5 (except as noted in
```

Comment:
```suggestion
   If there are no usable ServiceMode records, clients MUST go to step 5.  If all attempts fail, clients SHOULD go to step 5 (except as noted in
```

Reply:
"Usable" is ambiguous here.  I expanded this section and moved to our notion of "compatible" records (which may not have existed yet when we last updated this text).